File: src/main/java/fit/wenchao/http_file_server/model/vo/FileInfo.java
package fit.wenchao.http_file_server.model.vo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Accessors(chain = true)
public class FileInfo {
    String name;
    String path;
    long length;
    int type;
}

File: src/main/java/fit/wenchao/http_file_server/model/vo/FileInfo.kt
package fit.wenchao.http_file_server.model.vo;

data class FileInfo(
    var fileType: String = "file",
    var name: String = "",
    var path: String = "",
    var length: Long = 0,
    var type: Int = 0,
    var creationTime: String = "",
    var lastModifiedTime: String = "",
) {

}
